runtime/queries/rescript/locals.scm

```diff
@@ -1,7 +1,7 @@
 (switch_expression) @local.scope
-(if_expression) @local.scope
 
 ; Definitions
 ;------------
-(type_declaration) @local.defintion
-(let_binding) @local.defintion
+(type_declaration) @local.definition
+(let_binding) @local
+(module_declaration) @local.namespace
```

Comment:
```suggestion
(let_binding) @local.definition
(module_declaration) @local.definition
```

for locals we only support `local.scope`, `local.definition` and `local.reference` (following https://tree-sitter.github.io/tree-sitter/syntax-highlighting#local-variables)